crates/fluvio-hub-util/src/hubaccess.rs

```diff
@@ -92,15 +92,26 @@ impl HubAccess {
         self.get_action_auth(ACTION_PUBLISH).await
     }
 
+    pub async fn get_action_auth_with_token(
+        &self,
+        action: &str,
+        authn_token: &str,
+    ) -> Result<String> {
+        self.make_action_token(action, authn_token.into()).await
+    }
+
     async fn get_action_auth(&self, action: &str) -> Result<String> {
+        self.make_action_token(action, read_infinyon_token()?).await
+    }
```

Comment:
To avoid breaking the API we pass the Infinyon token under the hood using the same call to `make_action_token`